packages/ui/src/pages/app/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -111,7 +111,7 @@ export function SignInForm({
                           callbackUrl: `${location.origin}/redirecting`,
                         })
                       }
-                      className="w-full px-0 md:justify-start md:pl-20"
+                      className="w-full"
```

Comment:
Why do we need to remove other styles?

Reply:
The old style is not centered:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/43892874/214765234-be11ac80-a4cc-463a-81b8-5e3c53f3e631.png">
The new style(I also tested it on smaller screens):
<img width="500" alt="image" src="https://user-images.githubusercontent.com/43892874/214765332-6b22d762-111d-415a-bd28-2b8a406a9c14.png">

Reply:
Fixed by making icons align left.
<img width="600" alt="image" src="https://user-images.githubusercontent.com/43892874/215030673-6dda209f-5a1d-492d-9e11-13661709c6b2.png">